typegraph/node/sdk/src/runtimes/deno.ts

```diff
@@ -26,7 +26,7 @@ interface PredefinedFuncMat extends Materializer {
 }
 
 export interface DenoFunc {
-  code: string;
+  code: string | Function;
```

Comment:
The name `code` would suggest that it is a string...

Reply:
Yeah. right. I thought of renaming it but it will break old typegraphs that uses the deno feat.